lib/heex_formatter/phases/format.ex

```diff
@@ -29,8 +29,13 @@ defmodule HeexFormatter.Phases.Format do
   @default_line_length 98
 
   @spec run(list(), Keyword.t()) :: String.t()
-  def run(tokens, _opts) do
-    initial_state = %{html: "", previous_token: nil, indentation: 0}
+  def run(tokens, opts) do
+    initial_state = %{
+      html: "",
+      previous_token: nil,
+      indentation: 0,
+      mode: :html
+    }
```

Comment:
I would keep in one line for now, its not too much state keys yet

Reply:
I know :) but now my other pull request with line_length is merged, which makes it longer ;)